python/ray/rllib/policy_gradient/policy_gradient.py

```diff
@@ -213,21 +215,9 @@ def standardized(value):
         elif kl < 0.5 * config["kl_target"]:
             self.kl_coeff *= 0.5
 
-        checkpointing_start = time.time()
-        if config["model_checkpoint_file"]:
-            checkpoint_path = self.saver.save(
```

Comment:
So we used to checkpoint the model automatically at every iteration. Now the expectation is that people will call `algorithm.save` manually?

Reply:
I also added a `--checkpoint-freq` flag to train.py. It can be set to 1 or any other value.

Reply:
But yeah if you're using the python API then you checkpoint manually.

Reply:
ok, that makes sense